helm/thingsboard/templates/ingress.yaml

```diff
@@ -50,6 +50,13 @@ spec:
                 name: {{ $releaseName }}-http
                 port:
                   number: {{ $values.http.service.port }}
+          - path: /api
```

Comment:
I see very similar path just after this. Why is this one also needed? Could you please provide steps to reproduce the issue you are trying to fix?

Thank you!

Reply:
Despite seeing the issue multiple times when first installed, I have been unable to replicate it since.  It appears there could have been a long-running deploy job that was preventing the api path from being available for a while, and it was just a matter of waiting for it to finish starting (taking over an hour).  Since I am not still seeing this issue, I will close the request.